doc/specifications/proposed/field_type.md

```diff
@@ -0,0 +1,308 @@
+Field Type Creation Simplification
+==================================
+
+Overview
+--------
+
+First provide a couple of default implementations on the
+`eZ\Publish\Core\FieldType\FieldType` class. Omit all optional methods
```

Comment:
We should rather encourage implementing `eZ\Publish\SPI\FieldType\FieldType` as there is a BC promise on SPI.